### ISSUE TITLE
Show shared database files from google drive?

### ISSUE THREAD

**Issue by pwFoo** [keeweb/keeweb#832]
Shared databases can't be selected / not shown in google drive file selection?

Is it possible to make shared files available too?

**Comment by antelle** [comment]
Hi!
They're here: 
![drive](https://user-images.githubusercontent.com/633557/34469707-3cd9620c-ef25-11e7-95e3-b6cd18bacc26.png)
Aren't they shown for you?

**Comment by pwFoo** [comment]
Hi @antelle,

It's possible to add shared files to own folders. The shared file isn't shown inside of the folder. Keeweb desktop app not show all my folders, no "Show all files" and also no "Shared with me" folder.
Desktop app version is v1.6.3. No updates available.

Tested with app.keeweb.info "Show all files" and "Shared with me" are visible, but not the shared database file organized inside of my folder. But "Shared with me" with the desktop app would be fine for me.

**Comment by antelle** [comment]
I don't understand it. What is _shared files to own folders_?
What does it look like in Google Drive?

**Comment by antelle** [comment]
Example of shared folder:
![shared](https://user-images.githubusercontent.com/633557/34470318-4994ef20-ef2f-11e7-998e-b0d7fa35553a.png)
![keeweb](https://user-images.githubusercontent.com/633557/34470319-4bba5b3c-ef2f-11e7-86d9-56d33f05a532.png)
Are there other types of shared folders in Google Drive?

**Comment by pwFoo** [comment]
If you right click a shared file you can move it to a folder inside of your own My Drive folder.
Moved shared file not shown inside of the folder. Only inside of "Shared with me" (but would be ok).

My desktop app don't show "Shared with me"? Version is latest / up to date. So with desktop app I can't open shared files.

**Comment by antelle** [comment]
It looks like, you cannot move it, just copy: it remains visible in _Shared with me_ folder.
Do you see _Shared with me_ on app.keeweb.info? There's nothing special in desktop app, it should be the same.

**Comment by antelle** [comment]
What do you see in your Google Drive at all, are there any files, or folders? I mean, as seen by KeeWeb, of course.

**Comment by pwFoo** [comment]
After maximize the desktop app all folders visible! You need to check the screen size and optional add a scrollbar. Same with plugins gallery.

move / copy shared files is more like link / symlink a file. Rename the moved file also renames the shared file in "Shared with me". File is available in "Shared with me", but the "link" to my folder I moved it to isn't available as database.

Make links visible would be just a nice to have, but scrollbars would be important for usabliity.

**Comment by antelle** [comment]
Yeah, linked files should be implemented.
In plugins gallery, there is a scrollbar when you move the mouse over it. However, it's not visible when you click _Load plugin gallery_, only after you go to this page again. Should be fixed yeah.
On the first screen, it's disabled for simplicity for now, yeah, it should be there as well, but it's another issue of course.

**Comment by pwFoo** [comment]
Yes, me problem is solved by maximize the window and click at "Shared with me" in desktop app ;)
Thanks for helping me with my problem!

**Comment by antelle** [comment]
You can also scroll it as is, without maximizing, it could be a bit faster. I’ve fixed scrolling issue in plugins btw, thanks for mentioning it.

**Comment by antelle** [comment]
Just checked, Google Drive doesn't return links inside folders, so it won't be possible to implement unfortunately.
